sim/midas/src/main/cc/rtlsim/Makefrag-verilator

```diff
@@ -15,7 +15,30 @@
 #   (optional) verilator_conf: An verilator configuration file
 
 VERILATOR ?= verilator --cc --exe
-override VERILATOR_FLAGS := --assert -Wno-STMTDLY -O3 \
+
+# default flags added for ariane
+ARIANE_VERILATOR_FLAGS = \
+	--unroll-count 256 \
+	-Werror-PINMISSING \
+	-Werror-IMPLICIT \
+	-Wno-fatal \
+	-Wno-PINCONNECTEMPTY \
+	-Wno-ASSIGNDLY \
+	-Wno-DECLFILENAME \
+	-Wno-UNUSED \
+	-Wno-UNOPTFLAT \
+	-Wno-BLKANDNBLK \
+	-Wno-style \
+	-Wall
+
+# normal flags used for midas builds (that are incompatible with ariane)
+MIDAS_VERILATOR_FLAGS = \
+	--assert
+
+override VERILATOR_FLAGS := \
```

Comment:
If we're going to go as far as detecting what target is coming in from the top we should just emit this from the generator now.  We can put it in the generate makefrag which is already being sourced my the target-specific makefrag in sim/

Reply:
Discussed offline: As a middle ground, I moved this to the FireSim makefrag located in `sim/src/main/makefrag/firesim/Makefrag`